test/functional/explain.test.js

```diff
@@ -762,4 +763,41 @@ describe('Explain', function() {
       });
     })
   );
+
+  it('should throw a catchable error with invalid explain string (promise)', {
+    metadata: {
+      requires: {
+        mongodb: '>=3.4'
+      }
+    },
+    test: withClient(function(client, done) {
+      const db = client.db('shouldThrowCatchableError');
+      const collection = db.collection('test');
+      collection
+        .find({ a: 1 })
+        .explain('invalidExplain')
+        .catch(err => {
```

Comment:
can we add a `.then(() => done(new Error('expected explain to fail but it succeeded')))` or something along those lines here and in the test below, so that we fail early instead of a timeout in case this breaks?

also, for consistency, can we add a new line between these two `it` blocks?

Reply:
I think the callback test should fail immediately since the error and success are handled in a single callback, right? I've added the suggested early failure into the promise test.

Reply:
Yeah, you are right, I'm not sure what I was seeing originally, but the current version of the tests looks good